### DISCUSSION TITLE
Should we release 0.68.5?

### DISCUSSION THREAD

**Discussion by Titozzz** [reactwg/react-native-releases#37] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/45

---

Current Release: [0.68.4](https://github.com/facebook/react-native/releases/tag/v0.68.4)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by putuoka** [comment]
which version is the stable version of react native because it is very confusing

**Reply by sunnylqm** [reply]
"stable" is relative. A new patched version should always be more stable than old ones (fixed known issues). But there are always unknown issues, so keep an eye on updates. For newbies, my suggestions are:

1. Choose the latest version if you start a new project, downgrade only if you met issues that have no answers
2. Upgrade to latest patch version(like 0.68.0 -> 0.68.4, not 0.69 or 0.70) if you already have a project, unless the new features provided by new major version is a must.

Actually the above can apply to almost any software.

**Comment by retyui** [comment]
a small one but helpful: https://github.com/facebook/react-native/commit/475310dbbaec8048411edefc6cdddab330df7966

**Comment by cortinico** [comment]
Here the PR to include `exclusiveContent` on template for 0.68:
- https://github.com/facebook/react-native/pull/35218
